src/checkers/cvechecker.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 2018 Endless Mobile, Inc.
```

Comment:
You don't have to assign the copyright to Endless if you don't want and you're not using most of its code.
Also, please describe what this checker is about (see other checkers for examples).